app/pb_kits/playbook/pb_user/docs/example.yml

```diff
@@ -5,7 +5,10 @@ examples:
   - user_text_only: Text Only
   - user_size: Horizontal Size
   - user_vertical_size: Vertical Size
+  - user_dark: Dark
+
```

Comment:
Above you also created the doc file "_user_text_only_dark.html.erb" - this should be listed here as well